### PR TITLE
Prefixed Fullscreen API removed in Firefox 65

### PR DESCRIPTION
**Pull request by a2sheppy**
Updates the interfaces and CSS selector that are involved
in the Fullscreen API to note that the `moz` prefix is
gone in Firefox 65.

See bug 1504946.

Linting passed cleanly and visually things look good.

### CODE REVIEW

File: api/Element.json [hunk 1]
```diff
@@ -2874,7 +2888,7 @@
           }
         }
       },
-      "onfullscreenerror": {
+      "onerror": {
```
Comment (Elchi3) [inline api/Element.json]: drive by comment: why this change? onfullscreenerror -> onerror?

Comment (a2sheppy) [reply api/Element.json]: Accidental delete key press that I thought I caught but didn't. Fixing in an upcoming PR momentarily.

**Comment by foolip** [comment]
Sorry to bump an old issue, but I have discovered long after the fact that these changes were wrong. The moz-prefixed fullscreen APIs were *deprecated* in https://bugzilla.mozilla.org/show_bug.cgi?id=1504946 but not removed to this day.
